easybuild/easyconfigs/a/Amber/Amber-14-intel-2016a-AmberTools-15-patchlevel-13-13.eb

```diff
@@ -0,0 +1,41 @@
+#
+# modified by Stephane Thiell (Stanford University) for Amber v14
+#
+# author: Benjamin P. Roberts (University of Auckland)
+#
+# based on work by Marios Constantinou (University of Cyprus)
+#
+name = 'Amber'
+version = '14'
+
+homepage = 'http://ambermd.org/amber.html'
+description = """Amber (originally Assisted Model Building with Energy
+ Refinement) is software for performing molecular dynamics and structure
+ prediction."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'openmp': False, 'usempi': True}
+
+ambertools_ver = '15'
+sources = [
+    'Amber%(version)s.tar.bz2',
+    'AmberTools%s.tar.bz2' % ambertools_ver,
+]
+patches = [
+    'Amber-%(version)s_fix-hardcoding.patch',
+    'AmberTools-%s_fix-mdgx-print-bug.patch' % ambertools_ver,
+]
+
+dependencies = [
+    ('CUDA', '7.5.18', '', True),
+    ('netCDF', '4.4.0'),
+    ('netCDF-Fortran', '4.4.4'),
+    ('Python', '2.7.11'),
+]
+
+patchlevels = (13, 13)
+patchruns = 1
+
+versionsuffix = '-AmberTools-%s-patchlevel-%d-%d' % (ambertools_ver, patchlevels[0], patchlevels[1])
```

Comment:
using different patchlevels will result in a different build, since they determine how the source gets patched by the easyblock using the `update_amber` tool